volontulo/tests/views/test_offers.py

```diff
@@ -475,3 +475,165 @@ def test_for_correct_slug(self):
         self.assertIn('offer', response.context)
         self.assertIn('volunteers', response.context)
         self.assertEqual(len(response.context['volunteers']), 5)
+
+
+class TestOffersJoin(TestCase):
+    u"""Class responsible for testing offer's join page."""
+
+    @classmethod
+    def setUpTestData(cls):
+        u"""Set up data for all tests."""
```

Comment:
I prefer at least for now not to do this like that. Maybe, when we will deveop some kind of aggreement how Common class should looks like?